File: Webpage/server/webpage.server/webpage.server/src/main/java/abbruzzese/webpage/webpage/server/webpage/DTO/DTONewsletter.java
package abbruzzese.webpage.webpage.server.webpage.DTO;

public class DTONewsletter {
    
    private String email;

    public DTONewsletter(){}

    DTONewsletter(String Email){
        this.email=Email;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }
  
}

File: PHP_API/DTO/DTONewsletter.php
<?php

namespace App\DTO;

use JsonSerializable;

class DTONewsletter implements JsonSerializable{

    public $email;

    public function __construct($email) {
        $this->email = $email;
    }

    //Getters y Setters
    
    public function getEmail() {return $this->email;}
    public function setEmail($email) {$this->email = $email;}


    public function jsonSerialize(): array
    {
        return [
            'email' => $this->email,
        ];
    }
}

?>